spring-kafka/src/main/java/org/springframework/kafka/listener/DefaultAfterRollbackProcessor.java

```diff
@@ -137,20 +144,29 @@ public DefaultAfterRollbackProcessor(@Nullable
 	public DefaultAfterRollbackProcessor(@Nullable BiConsumer<ConsumerRecord<?, ?>, Exception> recoverer,
 			BackOff backOff, @Nullable BackOffHandler backOffHandler, @Nullable KafkaOperations<?, ?> kafkaOperations,
 			boolean commitRecovered) {
-
-		super(recoverer, backOff, backOffHandler);
+		super(recoverer, backOff, backOffHandler, new CommonErrorHandler() { });
 		this.kafkaTemplate = kafkaOperations;
 		super.setCommitRecovered(commitRecovered);
 		checkConfig();
 		this.backOff = backOff;
+		this.recoverer = (crs, ex) -> {
```

Comment:
it's invalid.
recoverer need transaction too
if use DLT,
need producer inTX, consumer inTX, DLT inTX, DLT consumer inTX